eng/Version.Details.xml

```diff
@@ -24,11 +24,11 @@
       <SourceBuild RepoName="command-line-api" ManagedOnly="true" />
     </Dependency>
     <!-- Necessary for source-build. This allows the live version of the package to be used by source-build. -->
-    <Dependency Name="System.IO.Pipelines" Version="7.0.0">
+    <Dependency Name="System.IO.Pipelines" Version="8.0.0">
       <Uri>https://github.com/dotnet/runtime</Uri>
       <Sha>d099f075e45d2aa6007a22b71b45a08758559f80</Sha>
```

Comment:
Don't you need to update the Sha entries too?

Reply:
Will update this after I get all the source build stuff corrected